scripts/start-db-in-docker.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -e -u -x
```

Comment:
It adds to the output for sure. This file was one that was removed in a previous commit, and I was re-committing it. It could stand to be improved a bit though.